### PR TITLE
S3 upload and new data locations

### PR DESCRIPTION
**Pull request by acron0**
Add ability to upload data to s3 and adjust ccm's default data sources

### CODE REVIEW

File: src-data/witan/models/upload_data.clj [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+(def default-bucket    "witan-data")
+(def default-profile   "witan")
+(def default-directory "datasets/default_datasets")
+(def default-folder    "witan.models.demography")
+(def filename-regex    #".csv")
+
+(defn upload-file
+  [dir bucket file]
+  (when (and (not (.isDirectory file))
```
Comment (mantree) [inline src-data/witan/models/upload_data.clj]: I'd consider moving this when logic to a filter before upload-file is called. Cause at the moment this function is really upload-file-if-its-not-a-dir-but-is-a-csv-file :)

File: src-data/witan/models/upload_data.clj [hunk 2]
```diff
@@ -0,0 +1,31 @@
+(ns witan.models.upload-data
+  (:require [clojure.string :as str]
+            [amazonica.aws.s3 :as aws]
+            [amazonica.aws.s3transfer :as aws-s3])
+  (:gen-class))
+
+(def default-bucket    "witan-data")
+(def default-profile   "witan")
```
Comment (mantree) [inline src-data/witan/models/upload_data.clj]: Not sure about the use of profiles here, will these get created on the box on start up? Normally just supply the region and have amazonica pick up the creds from the evironment

Comment (mantree) [inline src-data/witan/models/upload_data.clj]: Ah, read the comment above, this is just for local running, not prod stuff.

**Comment by tcoupland** [comment]
:shipit:

**Pull request merged**